src/mixins/date-mixin.ts

```diff
@@ -167,4 +167,36 @@ export default class DateMixin extends Vue {
 
     return `${dateStr} at ${timeStr} Pacific time`
   }
+
+  /**
+   * Add a number of months to a date and return "YYYY-MM-DD".
+   * Date must be in the format of "YYYY-MM-DD" and months is a number
+   * @example (3, 2023-02-03) -> "2023-05-03"
+   * @example (18, 2023-02-03) -> "2024-08-03"
+   */
+  addMonthsToDate (months: number, date: string): string {
+    let temp = this.yyyyMmDdToDate(date)
+    temp.setMonth(temp.getMonth() + Number(months))
```

Comment:
You shouldn't need to type `months` -- it's already a number.

Reply:
Good catch! I don't even know why I added that in the first place. I think I was testing something and I forgot to remove it.